rust/osm2lanes/src/transform.rs

```diff
@@ -485,3 +485,38 @@ fn osm_separation_type(x: &str) -> Option<BufferType> {
         _ => None,
     }
 }
+
+pub fn lanes_to_tags(lanes: &[LaneSpec], _cfg: &Config) -> Result<Tags, LaneSpecError> {
+    let mut tags = std::collections::BTreeMap::new();
+    tags.insert("highway".to_owned(), "yes".to_owned()); // TODO, what?
```

Comment:
Now there's a good question. :)   We don't use all the tags originally, so I think emitting a subset of them is reasonable. Except we _do_ use this one, only sometimes. I wonder if we're forced to stash it?